### DISCUSSION TITLE
Icons list

### DISCUSSION THREAD

**Discussion by sebsebseb1982** [blakeblackshear/frigate#15441] [Ask A Question] [answered]
Hi,

I saw in [doc](https://docs.frigate.video/configuration/reference/) that it is possible to choose an icon for a group of cameras ("LuCar" in the example). Where can I find the complete list of available icons?

Thank you,
Sébastien

**Comment by NickM-27** [comment, marked as answer]
The UI allows you to pick the icon

**Reply by sebsebseb1982** [reply]
Yep, it gives me the name of icon to put in conf (Ex: Car in UI --> LuCar in conf)

Thank you !

**Comment by gdcrocx** [comment]
Here is the list of icons that can be used - https://react-icons.github.io/react-icons/icons/lu/
